rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/ProjectCustomDataHost.kt

```diff
@@ -25,8 +25,11 @@ class ProjectCustomDataHost(project: Project) : LifetimedProjectComponent(projec
                 logger.info(item.key+" "+ item.newValueOpt)
                 ProjectUtil.focusProjectWindow(project, true)
                 project.solution.customData.data["UNITY_ActivateRider"] = "false";
-            }else if (item.key == "UNITY_Play" && item.newValueOpt!=null) {
-                play.set(item.newValueOpt!!.toBoolean())
+            }else if (item.key == "UNITY_Play") {
```

Comment:
let's avoid '!!' expression anywhere in our code. 
i.e. 
item.newValueOpt ?: error("newValueOpt should not be null")
...